lib/test-collection.js

```diff
@@ -178,19 +178,32 @@ class TestCollection extends EventEmitter {
 	_buildTests(tests) {
 		return tests.map(test => this._buildTestWithHooks(test));
 	}
+	_hasUnskippedTests() {
+		const getUnskippedCount = tests => tests.filter(test => {
+			return !(test.metadata && test.metadata.skipped === true);
+		}).length;
+		return (getUnskippedCount(this.tests.serial) + getUnskippedCount(this.tests.concurrent)) > 0;
```

Comment:
This would be clearer with [`Array#some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some).